### PR TITLE
Omit self in SW

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Following https://github.com/google/WebFundamentals/pull/4147, I've omitted `self` in Service Worker code snippets.

### CODE REVIEW

File: src/content/en/updates/2017/02/media-session.md [hunk 1]
```diff
@@ -307,7 +307,7 @@ browser can't fetch them. Here's how you could implement this:
 
     const FALLBACK_ARTWORK_URL = 'fallbackArtwork.png';
     
-    self.addEventListener('install', event => {
+    addEventListener('install', event => {
       self.skipWaiting();
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/media-session.md]: Should 'self' go away here too?

Comment (beaufortfrancois) [inline src/content/en/updates/2017/02/media-session.md]: I think it's best to omit only for top-level calls.

Comment (jpmedley) [inline src/content/en/updates/2017/02/media-session.md]: As an editor, I'll take your word for it. As a student of web development, I'd like to know why?

Comment (beaufortfrancois) [inline src/content/en/updates/2017/02/media-session.md]: > I guess it makes sense to use "self" when you want to be explicit about using the global, but that doesn't seem necessary if you're writing code at the top level anyway

**Pull request merged**